store/modules/forecastViz.js

```diff
@@ -21,7 +21,11 @@ export default moduleOptions => ({
     data: ['Current Truth', 'Truth As Of'],
     colours: Array(parseInt((moduleOptions.models.length)/10, 10)+1).fill(['#0d0887', '#46039f', '#7201a8', '#9c179e', '#bd3786', '#d8576b', '#ed7953', '#fb9f3a', '#fdca26', '#f0f921']).flat(),
     all_models: moduleOptions.all_models,
-    disclaimer: moduleOptions.disclaimer
+    disclaimer: moduleOptions.disclaimer || '',
```

Comment:
We think we don't need the default empty string here
```suggestion
    disclaimer: moduleOptions.disclaimer,
```